slices/coreutils.yaml

```
@@ -0,0 +1,121 @@
+package: coreutils
+
+slices:
+  bins:
+    essential:
+      - libacl1_libs
+      - libattr1_libs
+      - libc6_libs
+      - libcrypt1_libs
+      - libgcc-s1_libs
+      - libgmp10_libs
+      - libpcre2-8-0_libs
+      - libselinux1_libs
+    contents:
+      /bin/cat:
+      /bin/chgrp:
+      /bin/chmod:
+      /bin/chown:
+      /bin/cp:
+      /bin/date:
+      /bin/dd:
+      /bin/df:
+      /bin/dir:
+      /bin/echo:
+      /bin/false:
+      /bin/ln:
+      /bin/ls:
+      /bin/mkdir:
+      /bin/mknod:
+      /bin/mktemp:
+      /bin/mv:
+      /bin/pwd:
+      /bin/readlink:
+      /bin/rm:
+      /bin/rmdir:
+      /bin/sleep:
+      /bin/stty:
+      /bin/sync:
+      /bin/touch:
+      /bin/true:
+      /bin/uname:
+      /bin/vdir:
+      /usr/bin/[:
+      /usr/bin/arch:
+      /usr/bin/b2sum:
+      /usr/bin/base32:
+      /usr/bin/base64:
+      /usr/bin/basename:
+      /usr/bin/basenc:
+      /usr/bin/chcon:
+      /usr/bin/cksum:
+      /usr/bin/comm:
+      /usr/bin/csplit:
+      /usr/bin/cut:
+      /usr/bin/dircolors:
+      /usr/bin/dirname:
+      /usr/bin/du:
+      /usr/bin/env:
+      /usr/bin/expand:
+      /usr/bin/expr:
+      /usr/bin/factor:
+      /usr/bin/fmt:
+      /usr/bin/fold:
+      /usr/bin/groups:
+      /usr/bin/head:
+      /usr/bin/hostid:
+      /usr/bin/id:
+      /usr/bin/install:
+      /usr/bin/join:
+      /usr/bin/link:
+      /usr/bin/logname:
+      /usr/bin/md5sum:
+      /usr/bin/md5sum.textutils: { symlink: /usr/bin/md5sum }
```

SInce this already comes as a symlink from the package, you don't need to manually create it.
```suggestion
      /usr/bin/md5sum.textutils:
```

---

Updated, thanks!